src/DEvahebLibTests/BasicTests/dowait.txt

```diff
@@ -1,6 +1,7 @@
 //Generated by BehavEd
 
 dowait ( "Do Wait Combo String" );
+dowait ( "dont_shoot" );

```

Comment:
I'll have to look at this code when I get home, it's been a while. But why does the above example ("Do Wait Combo String") work fine?

Thanks for updating tests, by the way.

Reply:
I'm afraid it's a case-sensitive replacement, so the capital D causes it to not trigger...

A real-life example would be one of the `warden`-related scripts in `artus_detention` in Jedi Outcast.

Reply:
🤦‍♂️ oh great

Reply:
At some point I'd worked on running a whole game through the tests but some of the source files from SDKs were clearly not just edited with the existing tools meaning comparing source and binary didn't match... but not because of the tool.